tools/assign_missing_instruments.php

```diff
@@ -138,15 +137,15 @@ function populateVisitLabel($result, $visit_label)
     $where = array('vl' => $argv[1]);
 
     $results = $DB->pselect($query, $where);
-    foreach ($results AS $result) {
+    foreach ($results as $result) {
         populateVisitLabel($result, $visit_label);
     }
-} else if (isset($visit_labels)) {
+} elseif (isset($visit_labels)) {
```

Comment:
stylistically I think this change is bad, what rule is requiring it?

Reply:
It's not required actually. Not sure why this change was made. I'll revert.